src/market.ts

```diff
@@ -538,6 +538,7 @@ function handleOrderCreated(
 
   // Record the final collateral delta for the order
   order.collateral = order.collateral.plus(finalCollateralDelta)
+  order.endCollateral = order.startCollateral.plus(order.collateral)
```

Comment:
I'm wondering if this value is more interesting when including PNL and fees (so we'd also update in in the accountpositionprocess), or how were you imagining this being used?

Reply:
Before the account is settled and collateral adjusted for funding/fees, the user must ensure maintenance/margin requirments are satisfied by their pending order, and likely has their own target collateralization.  This metric provides this naive estimate of how much collateral will be backing a position change made by the order.